server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaBuilder.java

```diff
@@ -357,9 +390,10 @@ private CentralDogmaConfig buildConfig() {
 
         return new CentralDogmaConfig(dataDir, ports, tls, numWorkers, maxNumConnections,
                                       requestTimeoutMillis, idleTimeoutMillis, maxFrameLength,
-                                      numRepositoryWorkers, cacheSpec, gracefulShutdownTimeout,
-                                      webAppEnabled, webAppSessionTimeoutMillis,
-                                      mirroringEnabled, numMirroringThreads,
+                                      // TODO(minwoox) remove one repositoryCacheSpec
+                                      numRepositoryWorkers, repositoryCacheSpec, repositoryCacheSpec,
```

Comment:
It might be OK to replace the second `repositoryCacheSpec` with `null`, isn't it?